### PR TITLE
Update links to examples

### PR DESCRIPTION
**Pull request by mprse**
Update links to the following examples:
- Platform,
- IO,
- RTOS,
- NFC,
- Network,
- Storage.

Related PR that brings example to https://github.com/ARMmbed/mbed-os-examples-docs_only: 
- https://github.com/ARMmbed/mbed-os-examples-docs_only/pull/70
- https://github.com/ARMmbed/mbed-os-examples-docs_only/pull/71
- https://github.com/ARMmbed/mbed-os-examples-docs_only/pull/75
- https://github.com/ARMmbed/mbed-os-examples-docs_only/pull/80

@AnotherButler Please review. Not sure if this is done in the correct way. I think we should wait with this one until the above list is merged.

### CODE REVIEW

**Comment by jamesbeyond** [comment]
I believe all the code dependency been merged. could you review this PR @AnotherButler

**Comment by AnotherButler** [comment]
For cellular, do we use the one in this PR or the official example, which should be kept up to date: https://github.com/ARMmbed/mbed-os-example-cellular/blob/mbed-os-5.15/main.cpp? @adbridge @jamesbeyond

**Comment by adbridge** [comment]
@AnotherButler I think this is a question for @AnttiKauppila ?

**Review by jamesbeyond — APPROVED** [review]

**Comment by jamesbeyond** [comment]
> For cellular, do we use the one in this PR or the official example, which should be kept up to date: https://github.com/ARMmbed/mbed-os-example-cellular/blob/mbed-os-5.15/main.cpp? @adbridge @jamesbeyond


the content of 
`https://github.com/ARMmbed/mbed-os-examples-docs_only/blob/master/APIs_NetworkInterfaces/Cellular/main.cpp` 
and 
`https://github.com/ARMmbed/mbed-os-example-cellular/blob/mbed-os-5.15/main.cpp` 
are identical. 


I agree to keep one, maybe `https://github.com/ARMmbed/mbed-os-example-cellular/blob/mbed-os-5.15/main.cpp` would be better

File: docs/api/networkinterfaces/CellularInterface.md [hunk 1]
```diff
@@ -65,7 +65,7 @@ To bring up the network interface:
 
 This example establishes connection with the cellular network using Mbed OS CellularInterface.
 
-[![View code](https://www.mbed.com/embed/?url=https://github.com/ARMmbed/mbed-os-example-cellular)](https://github.com/ARMmbed/mbed-os-example-cellular/blob/mbed-os-5.14/main.cpp)
+[![View code](https://www.mbed.com/embed/?url=github.com/ARMmbed/mbed-os-examples-docs_only/blob/master/APIs_NetworkInterfaces/Cellular/)](github.com/ARMmbed/mbed-os-examples-docs_only/blob/master/APIs_NetworkInterfaces/Cellular/main.cpp)
```
Comment (jamesbeyond) [inline docs/api/networkinterfaces/CellularInterface.md]: @mprse could you change the link pointing to https://github.com/ARMmbed/mbed-os-example-cellular/blob/master/main.cpp

Don't worries about https://github.com/ARMmbed/mbed-os-examples-docs_only/tree/master/APIs_NetworkInterfaces/Cellular, I'll create a PR to remove that

**Review by jamesbeyond — CHANGES_REQUESTED** [review]

File: docs/api/networkinterfaces/CellularInterface.md (hunk 1 above)
Comment (jamesbeyond) [inline docs/api/networkinterfaces/CellularInterface.md]:
```suggestion
[![View code](https://www.mbed.com/embed/?url=https://github.com/ARMmbed/mbed-os-example-cellular/tree/master)](https://github.com/ARMmbed/mbed-os-example-cellular/blob/master/main.cpp)
```

**Comment by jamesbeyond** [comment]
Hi @AnotherButler, I updated this link, this PR is good to go

**Review by jamesbeyond — APPROVED** [review]

**Review by AnotherButler — APPROVED** [review]
Thanks for the PR 👍

**Pull request merged**